pr2eus/robot-interface.l

```diff
@@ -401,7 +401,15 @@
    av)
   (:angle-vector-sequence
    (avs &optional (tms (list 3000)) (ctype controller-type) (start-time 0.1) &key (scale 1) (min-time 0.0) (end-coords-interpolation nil))
-   "Send joind angle to robot, this method retuns immediately, so use :wait-interpolation to block until the motion stops."
+   "Send joind angle to robot, this method retuns immediately, so use :wait-interpolation to block until the motion stops.
+- asv : list of joint angle vector [rad]
+- tms : list duration from previous angle-vector to next goal in [msec]
+- ctype : controller method name
+- start-time : time to start moving
+- scale : if tm is not specified, it will use 1/scale of the fastest speed
+- min-time : minimum time to for time to goal
```

Comment:
Is this for `:angle-vector`?
How about `:angle-vector-sequence`?
And, `minimum time to for time to goal` -> `minimum time to goal` in english??